docs/topics/kotlin-osgi.md

```diff
@@ -94,7 +94,7 @@ dependencies {
 ### Why not just add required manifest options to all Kotlin libraries
 
 Even though it is the most preferred way to provide OSGi support, unfortunately it couldn't be done for now due to so called
-["package split" issue](http://wiki.osgi.org/wiki/Split_Packages) that couldn't be easily eliminated and such a big change is
```

Comment:
No idea how to replace it, please help.

Reply:
Here's an option of a URL. This points to a release note explaining what are OSGi split package: https://docs.osgi.org/specification/osgi.core/7.0.0/framework.module.html#d0e5999